lib/elastic/configuration.rb

```diff
@@ -14,8 +14,19 @@ class Configuration
       disable_index_name_caching: false
     }
 
-    attr_accessor :host, :port, :api_client, :index, :page_size, :coord_similarity, :logger,
-      :import_batch_size, :whiny_indices, :time_zone, :disable_indexing, :disable_index_name_caching
+    attr_accessor :host,
+                  :port,
+                  :adapter,
+                  :api_client,
+                  :index,
+                  :page_size,
+                  :coord_similarity,
+                  :logger,
+                  :import_batch_size,
+                  :whiny_indices,
+                  :time_zone,
+                  :disable_indexing,
+                  :disable_index_name_caching
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.